### ISSUE TITLE
[BITV] 9.4.1.2/9.1 - The navigation menu (tree element) implementation is incorrect. The buttons to open the nodes have been implemented as button elements that are inside links.

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/documentation#9746]
This should be changed. The aria-expanded attribute should be used with button elements and not on ul and li elements. For more information, see https://www.w3.org/TR/wai-aria-1.2/#aria-expanded and https://www.w3.org/WAI/ARIA/apg/patterns/treeview/. (1)

- [ ] Fix
- [ ] Backport

<img width="504" alt="df5487b90d1bb75c07396787724516b0" src="https://user-images.githubusercontent.com/1315170/223754854-76bc9fab-eadb-4147-962b-58fdbc12a40e.png">

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v9-n1

</details>
